src/Compilers/Core/Portable/DiagnosticAnalyzer/AsyncQueue.cs

```diff
@@ -52,7 +52,7 @@ public void Enqueue(TElement value)
         {
             if (!EnqueueCore(value))
             {
-                throw new InvalidOperationException($"Cannot call {nameof(Enqueue)} when the queue is already completed.");
+                throw new InvalidOperationException(string.Format(CodeAnalysisResources.CannotCallWhenQueueIsCompleted, nameof(Enqueue)));
```

Comment:
Alternatively, we could use distinct messages that do not require formatting.

---
In reply to: [114189608](https://github.com/dotnet/roslyn/pull/18901#discussion_r114189608) [](ancestors = 114189608)